src/api/entities/__tests__/Checkpoint.ts

```diff
@@ -216,13 +216,7 @@ describe('Checkpoint class', () => {
         returnValue: [],
       });
 
-      const assetBalance = new BigNumber(10);
-
-      entityMockUtils.configureMocks({
-        identityOptions: {
-          getAssetBalance: assetBalance,
-        },
-      });
+      const assetBalance = new BigNumber(1000);
```

Comment:
I am not sure why this changed, the old setup doesn't seem to work, as in it still returns `1000` now.

Reply:
I'm not sure I understand

Reply:
The old setup of "10" wasn't working, but now Identity is coming from a different call, so now I see why its happening.